SparkPost/support-docs
# WE-320: support docs home page

### What Changed
Adds the support docs homepage

### How to test
- navigate to 'localhost:3000/docs'
- make sure page matches [design](https://www.figma.com/proto/RPSbLfkkxLY5HKY32tjr8X/Support-Docs-Homepage?node-id=68%3A3599&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=68%3A3599)
